lib/linter.coffee

```diff
@@ -0,0 +1,147 @@
+{BufferedProcess, CompositeDisposable} = require 'atom'
+helpers = require 'atom-linter'
+fs = require 'fs'
+fse = require 'fs-extra'
+path = require 'path'
+temp = require 'temp'
+XRegExp = require('xregexp').XRegExp
```

Comment:
We also have a `rangeFromLineNumber` method here https://github.com/AtomLinter/atom-linter/blob/master/lib/helpers.coffee#L46